integration/tests/economy/test_economics.py

```diff
@@ -1268,43 +1423,49 @@ def test_multiple_scheduled_trx_with_failed_trx(
     def test_scheduled_trx_for_erc20_for_spl(
         self,
         web3_client_sol,
-        neon_user,
+        neon_user_with_all_tokens,
```

Comment:
please leave as it was with neon_user
and add a new test with neon user with sols inside neons (when there is enough sols and less then needed for the trx)